### ISSUE TITLE
Add ignore_index option to Accuracy metric

### ISSUE THREAD

**Issue by stas-sl** [PyTorchLightning/pytorch-lightning#6448]
## 🚀 Feature

It would be nice if `Accuracy` metric accepted `ignore_index` as it works for `StatsScore` or `torch.nn.CrossEntropyLoss`
